src/main/java/com/doit/api/Item/ItemController.java

```diff
@@ -0,0 +1,44 @@
+package com.doit.api.Item;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/items")
+public class ItemController {
+    private ItemService itemService;
+
+    @Autowired
+    public ItemController(ItemService itemService) {
+        this.itemService = itemService;
+    }
+
+    @PostMapping(path = "/post")
```

Comment:
When we use the Http method `POST` it is redundant to add `/post` to the resource's url.